examples/async_entity_tx_timed.rs

```diff
@@ -33,45 +34,38 @@ async fn main() {
         .await
         .unwrap();
 
-    let edn_body = client
+    let entity_tx_body = client
         .entity_tx_timed(person1.crux__db___id, None, Some(timed))
         .await
         .unwrap();
 
-    println!("\n Edn Body = {:#?}", edn_body);
-    // Edn Body = Map(
-    //     Map(
-    //         {
-    //             ":crux.db/id": Key(
-    //                 ":hello-entity",
-    //             ),
-    //             ":first-name": Str(
-    //                 "Hello",
-    //             ),
-    //             ":last-name": Str(
-    //                 "World",
-    //             ),
-    //         },
-    //     ),
-    // )
+    return entity_tx_body;
 }
 
-ser_struct! {
-    #[derive(Debug, Clone)]
-    #[allow(non_snake_case)]
-    pub struct Person {
-        crux__db___id: CruxId,
-        first_name: String,
-        last_name: String
-    }
+#[tokio::main]
+async fn main() {
+    let entity = entity_tx_timed().await;
+    println!("\n Edn Body = {:#?}", entity);
+    // Edn Body = EntityTxResponse {
+    //     db___id: "f936408359776345394b07809bf1fd9bf0f70046",
+    //     db___content_hash: "621f30a89898d2c55bc81b0b1e0db0be2878486c",
+    //     db___valid_time: 2014-11-29T06:00:09+00:00,
+    //     tx___tx_id: 111,
+    //     tx___tx_time: 2020-09-30T13:22:02.795+00:00,
+    // }
+}
+
+#[tokio::test]
+async fn test_entity_tx_timed() {
+    let entity = entity_tx_timed().await;
+
+    assert!(entity.tx___tx_id > 0);
```

Comment:
I didn't get this test, why checking if the `id` is bigger than zero?

Is it to check if its not empty? Maybe use the `is_empty` instead.

Reply:
any test works here, because it is mostly to detect if there is no inconsistencies with date, not really going to change to is_empty sorry